src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionCommentsIndentationTest.java

```diff
@@ -0,0 +1,219 @@
+                expectedXpathQueries);
+    }
+
+    @Test
+    public void testFive() throws Exception {
+        final File fileToProcess =
+                new File(getPath("SuppressionXpathRegressionCommentsIndentationFive.java"));
+
+        final DefaultConfiguration moduleConfig =
+                createModuleConfig(CommentsIndentationCheck.class);
+
+        final String[] expectedViolation = {
+            "6:1: " + getCheckMessage(CommentsIndentationCheck.class,
+                "comments.indentation.single", 7, 0, 4),
+        };
+
+        final List<String> expectedXpathQueries = Collections.singletonList(
+            "/CLASS_DEF[./IDENT[@text='SuppressionXpathRegressionCommentsIndentationFive']]"
+                + "/OBJBLOCK/METHOD_DEF[./IDENT[@text='foo']]/SLIST/SINGLE_LINE_COMMENT[2]"
+        );
+
+        runVerifications(moduleConfig, fileToProcess, expectedViolation,
+                expectedXpathQueries);
+    }
+
+    @Test
+    public void testSix() throws Exception {
+        final File fileToProcess =
+                new File(getPath("SuppressionXpathRegressionCommentsIndentationSix.java"));
+
+        final DefaultConfiguration moduleConfig =
+                createModuleConfig(CommentsIndentationCheck.class);
+
+        final String[] expectedViolation = {
+            "10:20: " + getCheckMessage(CommentsIndentationCheck.class,
+                "comments.indentation.single", "9, 11", 19, "16, 12"),
+        };
+
+        final List<String> expectedXpathQueries = Collections.singletonList(
+            "/CLASS_DEF[./IDENT[@text='SuppressionXpathRegressionCommentsIndentationSix']]"
+                + "/OBJBLOCK/METHOD_DEF[./IDENT[@text='foo']]/SLIST/LITERAL_SWITCH/"
+                + "CASE_GROUP/SINGLE_LINE_COMMENT"
+        );
+
+        runVerifications(moduleConfig, fileToProcess, expectedViolation,
+                expectedXpathQueries);
+    }
+
+    @Test
+    public void testSeven() throws Exception {
+        final File fileToProcess =
+                new File(getPath("SuppressionXpathRegressionCommentsIndentationSeven.java"));
+
+        final DefaultConfiguration moduleConfig =
+                createModuleConfig(CommentsIndentationCheck.class);
+
+        final String[] expectedViolation = {
+            "9:1: " + getCheckMessage(CommentsIndentationCheck.class,
+                "comments.indentation.single", "8, 10", 0, "12, 12"),
+        };
+
+        final List<String> expectedXpathQueries = Collections.singletonList(
+            "/CLASS_DEF[./IDENT[@text='SuppressionXpathRegressionCommentsIndentationSeven']]"
+                + "/OBJBLOCK/METHOD_DEF[./IDENT[@text='foo']]/SLIST/LITERAL_SWITCH/"
+                + "CASE_GROUP/SINGLE_LINE_COMMENT"
+        );
+
+        runVerifications(moduleConfig, fileToProcess, expectedViolation,
+                expectedXpathQueries);
+    }
+
+    @Test
+    public void testEight() throws Exception {
```

Comment:
please change names of tests and test inputs to something more meaningful, because they all seem the same